### PR TITLE
Move database structures to indexer-search

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: worker/store/store.go [hunk 1]
```diff
@@ -3,15 +3,21 @@ package store
...
 }
 
 type HeightStore interface {
 	StoreTransactions(ctx context.Context, txs []structs.TransactionWithMeta) error
 	StoreBlocks(ctx context.Context, blocks []structs.BlockWithMeta) error
 	ConfirmHeights(ctx context.Context, heights []structs.BlockWithMeta) error
 }
+
+type RewardStore interface {
+	StoreRewards(ctx context.Context) error
```
Comment (pacmessica) [inline worker/store/store.go:22]: Do we need this here? I think we want to move all reward logic to indexer-rewards (including db storage), so not sure what this is for

File: worker/store/transport/reward.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"context"
+
+	"github.com/figment-networks/indexer-rewards/structs"
+)
+
+type RewardStore struct {
+	*HTTPStoreSession
+}
+
+func (s *HeightStore) StoreClaimedRewards(ctx context.Context, rewards []structs.ClaimedReward) error {
```
Comment (pacmessica) [inline worker/store/transport/reward.go:13]: I'm not sure about this - not as familiar with the new manager arch as you guys. Is this the right place for this? @lukanus

**Review by pacmessica — COMMENTED** [review]
made a comment on [this pr](https://github.com/figment-networks/indexer-search/pull/4) about moving structs/utils over, so same applies here.

otherwise looks straight forward. Just had some questions about the store reward calls

File: worker/store/store.go (hunk 1 above)
Comment (patiee) [reply worker/store/store.go]: good idea, I'll move it to indexer-rewards

**Pull request merged**